clients/python/sust/api/climate_explorer/__init__.py

```diff
@@ -0,0 +1 @@
+from .client import Client
```

Comment:
I don't typically do this, but I do think it is convenient to export types here that are very commonly used. Feedback appreciated on this